### DISCUSSION TITLE
Auto updates

### DISCUSSION THREAD

**Discussion by noideas4thisname** [Nimdy/Dedicated_Valheim_Server_Script#77] [Q&A] [answered]
It's unclear to me the behavior around updates. In the readme there is a note about how the server updates on its own on restart (great) - but is there a mechanism in place to trigger restart upon release of a new patch?

Curious mostly because I went to update the server today and it seemed it was already the latest version - but there was a brief period of time players weren't able to connect.

In either case this is a nifty little script that makes the job easy - so thank you to the developer!

**Comment by Nimdy** [comment, marked as answer]
Thanks! Its a team effort at this point :)

The server will automatically update on restart of services or the server reboots.  This is by IronGates design but maybe we just take that out and have people do it manually?

I dont know.. yet... not a lot of people understand Linux, so this script is aimed to help me get excited about and start learning a little...just by messing around with the menu system and being forced to play in the GUI for simple stuff.

Thanks again!

**Reply by noideas4thisname** [reply]
I'd love for detection of an update to trigger the server to restart. To me an always up-to-date server outweighs the cons that could bring for the small audience I serve, and that seems completely do-able (i.e. check for updates in X interval, if found restart the service). I may look into this if I have some downtime as well.
